### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by pyhys**

### CODE REVIEW

File: src/common/pf_alarm.c [hunk 1]
```diff
@@ -1125,13 +1135,16 @@ static int pf_alarm_apms_a_data_req (
...
+         PF_AL_BUF_LOG,
+         "Alarm(%d): Allocate RTA output buffer\n",
+         __LINE__);
       p_rta = os_buf_alloc (PF_FRAME_BUFFER_SIZE);
       if (p_rta == NULL)
       {
          LOG_ERROR (
             PF_ALARM_LOG,
-            "Alarm(%d): No buffer for alarm notification\n",
+            "Alarm(%d): No output buffer for alarmn",
```
Comment (olbjo) [inline src/common/pf_alarm.c:1147]: typo
```suggestion
            "Alarm(%d): No output buffer for alarm",
```

File: src/device/pf_diag.c [hunk 2]
```diff
@@ -311,6 +328,16 @@ int pf_diag_add (
             }
             else
             {
+               LOG_DEBUG (
+                  PNET_LOG,
```
Comment (olbjo) [inline src/device/pf_diag.c:332]: Maybe a separate define for diag/alarms?

File: src/pf_types.h [hunk 3]
```diff
@@ -2025,7 +2025,7 @@ typedef struct pf_diag_item
 
    bool in_use;
    uint16_t usi;  /* pf_usi_values_t */
-   uint16_t next; /* Next in list */
+   uint16_t next; /* Index (into array) of next in list */
```
Comment (olbjo) [inline src/pf_types.h:2028]:
```suggestion
   uint16_t next; /* Next in list (array index) */
```

**Review by olbjo — APPROVED** [review]
Minor comments only

File: src/common/pf_alarm.c (hunk 1 above)
Comment (pyhys) [reply src/common/pf_alarm.c]: Done

File: src/device/pf_diag.c (hunk 2 above)
Comment (pyhys) [reply src/device/pf_diag.c]: Used PF_ALARM_LOG instead.
Done

File: src/pf_types.h (hunk 3 above)
Comment (pyhys) [reply src/pf_types.h]: Done

**Pull request merged**